src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -396,7 +396,7 @@ public function hardlink(string $originFile, $targetFiles)
     private function linkException(string $origin, string $target, string $linkType)
     {
         if (self::$lastError) {
-            if ('\\' === \DIRECTORY_SEPARATOR && str_contains(self::$lastError, 'error code(1314)')) {
+            if ('\\' === \DIRECTORY_SEPARATOR && str_contains((string) self::$lastError, 'error code(1314)')) {
```

Comment:
this change is wrong. It cannot be null here due to the surrounding `if (self::$lastError)`